src/SFA.DAS.FindApprenticeshipTraining/Services/LocationApiClient.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
```

Comment:
I'm assuming that all the `NotImplemented` is because we don't need those calls right now?